package.json

```diff
@@ -10,19 +10,18 @@
     "url": "https://github.com/preactjs/preact-www.git"
   },
   "scripts": {
-    "dev": "concurrently 'npm run dev:client' 'npm run dev:lambda'",
+    "dev": "concurrently \"npm run dev:client\" \"npm run dev:lambda\"",
     "dev:client": "preact watch --template src/template.html",
     "dev:lambda": "netlify-lambda serve src/lambda",
     "start": "npm run -s server",
     "prestart": "npm run -s build",
     "server": "superstatic build -p ${PORT:-8080} --host 0.0.0.0 --gzip -c '{\"rewrites\": [{\"source\":\"**\",\"destination\":\"index.html\"}],\"headers\":[{\"source\":\"**\",\"headers\":[{\"key\":\"Cache-Control\",\"value\":\"max-age=31536000\"}]}]}'",
     "build": "netlify-lambda build src/lambda && preact build --prerenderUrls src/prerender.js --template src/template.html",
-    "postbuild": "cat src/_headers >> build/_headers",
```

Comment:
Unfortunately `cat` is meaningless in Windows cmd :/  So I moved this into preact.config.js 🤷‍♂️ Open to other suggestions